addons/radar/XEH_preInit.sqf

```diff
@@ -80,6 +80,29 @@ if (_acre || _tfar) then {
     GVAR(showSpeaking_replaceIcon) = false;
 };
 
+[
+  "diwako_dui_toggleAceMedic",
```

Comment:
Use GVAR macro instead of spelled out string. It is a bit confusing i do admit, it is because of backwards compatibility before this mod was a CBA template mod.

New CBA settings should take advantage of the CBA macros